spec/shopify_transporter/exporters/magento/customer_exporter_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+require 'shopify_transporter/pipeline/stage'
+
+RSpec.describe ShopifyTransporter::Exporters::Magento::CustomerExporter do
+  context '#run' do
+    it 'retrieves customers from Magento using the SOAP API and returns the results' do
+      soap_client = double("soap client")
+
+      customer_customer_list_response_body = double('customer_customer_list_response_body')
+      customer_address_list_response_body = double('customer_address_list_response_body')
+
+      expect(soap_client)
+        .to receive(:call)
+        .with(:customer_customer_list, anything)
+        .and_return(customer_customer_list_response_body).at_least(:once)
+
+      expect(customer_customer_list_response_body).to receive(:body).and_return(
+        {
+          customer_customer_list_response: {
+            store_view: {
+              item: [
+                {
+                  customer_id: 654321,
+                  top_level_attribute: "an_attribute"
+                }
+              ]
+            }
+          }
+        }
+      ).at_least(:once)
+
+      expect(soap_client)
+        .to receive(:call)
+        .with(:customer_address_list, customer_id: 654321)
+        .and_return(customer_address_list_response_body).at_least(:once)
+
+      expect(customer_address_list_response_body).to receive(:body).and_return(
+        {
+          customer_address_list_response: {
+            result: {
+              customer_address_attribute: "another_attribute"
+            }
+          }
+        }
+      ).at_least(:once)
+
+      expected_result = [
+        {
+          customer_id: 654321,
+          top_level_attribute: "an_attribute",
+          address_list: {
+            customer_address_attribute: "another_attribute",
+          }
+        }
+      ]
+
+      exporter = ShopifyTransporter::Exporters::Magento::CustomerExporter.new(1, soap_client)
```

Comment:
What do you think of the idea of making the initializer parameter named?  Would make it easier to determine what `1` is.